sdk/core/Azure.Core/tests/FailedResponseExceptionTests.cs

```diff
@@ -165,6 +167,22 @@ public async Task IncludesInnerException()
             Assert.AreEqual(innerException, exception.InnerException);
         }
 
+        [Test]
+        public void RequestFailedExceptionIsSerializeable()
+        {
+            var dataContractSerializer = new DataContractSerializer(typeof(RequestFailedException));
+            var exception = new RequestFailedException(201, "Message", "Error", null);
```

Comment:
(extreme) nit: You may want to use a non-200 series response code, just for clarity.